Composer/packages/tools/language-servers/language-generation/src/lgParser.ts

```diff
@@ -20,8 +23,10 @@ export class LgParser {
   private rejects = {};
 
   constructor() {
-    const workerScriptPath = path.join(__dirname, 'lgWorker.js');
-    this.worker = fork(workerScriptPath);
+    const fileName = isDev ? 'lgWorker.ts' : 'lgWorker.js';
+    const execArgv = isDev ? ['-r', 'ts-node/register'] : [];
```

Comment:
this should only need to be done in the test environment, in the dev environment we should expect all the TypeScript to be compiled

Reply:
After re-reading this I see that the `start` script in the `language-generation` project uses `ts-node` to start the server without compiling the TypeScript.

When the electron app is started in the dev environment, we set the node environment to `"development"` so I'm curious to see if this code would still work and execute the TypeScript.

Did you test this when running the electron app from source?

Reply:
![image](https://user-images.githubusercontent.com/3452012/84926798-c70c5c80-b080-11ea-9124-ec9f04045f77.png)

This breaks when running the electron app from source

Reply:
can the `start` script in `/language-generation/package.json` just call `build` first and then call `node /lib/server.js`?

Reply:
can the `start` script for this package just compile the TypeScript in the `/demo/` folder and then call `node /demo/dist/server.js` ?

Reply:
Hi, @tonyanziano The demo is used to test, I think I can just set the node_env=test